### ISSUE TITLE
netdata-uninstaller.sh not working (with macOS 10.13)

### ISSUE THREAD

**Issue by maurice-schleussinger** [firehol/netdata#2941]
I can't uninstall netdata with the `netdata-uninstaller.sh` script as it uses commands which are not available under macOS 10.13 (or any version of macOS?)

```
└─[$] ./netdata-uninstaller.sh --force                                                                                                                                                             [12:19:41]
Stopping a possibly running netdata...
./netdata-uninstaller.sh: line 13: pidof: command not found

Deleting directory '/Users/someone/opt/netdata' ...
rm: illegal option -- I
usage: rm [-f | -i] [-dPRrvW] file ...
       unlink file
./netdata-uninstaller.sh: line 78: getent: command not found

You may also want to remove the user netdata
by running:
   userdel netdata
./netdata-uninstaller.sh: line 87: getent: command not found

You may also want to remove the group netdata
by running:
   groupdel netdata
./netdata-uninstaller.sh: line 96: getent: command not found
./netdata-uninstaller.sh: line 105: getent: command not found
./netdata-uninstaller.sh: line 114: getent: command not found
./netdata-uninstaller.sh: line 123: getent: command not found
./netdata-uninstaller.sh: line 132: getent: command not found
./netdata-uninstaller.sh: line 141: getent: command not found
./netdata-uninstaller.sh: line 150: getent: command not found
./netdata-uninstaller.sh: line 159: getent: command not found
```

**Comment by quadcube** [comment]
same issue. (macOS v10.13.2).
`./netdata-uninstaller.sh: line 13: pidof: command not found

Deleting directory '/Users/quadcube/opt/netdata' ...
rm: illegal option -- I
usage: rm [-f | -i] [-dPRrvW] file ...
       unlink file
./netdata-uninstaller.sh: line 78: getent: command not found

You may also want to remove the user netdata
by running:
   userdel netdata
./netdata-uninstaller.sh: line 87: getent: command not found

You may also want to remove the group netdata
by running:
   groupdel netdata
./netdata-uninstaller.sh: line 96: getent: command not found
./netdata-uninstaller.sh: line 105: getent: command not found
./netdata-uninstaller.sh: line 114: getent: command not found
./netdata-uninstaller.sh: line 123: getent: command not found
./netdata-uninstaller.sh: line 132: getent: command not found
./netdata-uninstaller.sh: line 141: getent: command not found
./netdata-uninstaller.sh: line 150: getent: command not found
./netdata-uninstaller.sh: line 159: getent: command not found`

**Comment by ferazambuja** [comment]
I have the same issue.
```
sudo ./netdata-uninstaller.sh --force
Stopping a possibly running netdata...
./netdata-uninstaller.sh: line 13: pidof: command not found

Deleting directory '/usr/local/netdata' ...
rm: illegal option -- I
usage: rm [-f | -i] [-dPRrvW] file ...
       unlink file
./netdata-uninstaller.sh: line 78: getent: command not found

You may also want to remove the user netdata
by running:
   userdel netdata
./netdata-uninstaller.sh: line 87: getent: command not found

You may also want to remove the group netdata
by running:
   groupdel netdata
./netdata-uninstaller.sh: line 96: getent: command not found
./netdata-uninstaller.sh: line 105: getent: command not found
./netdata-uninstaller.sh: line 114: getent: command not found
./netdata-uninstaller.sh: line 123: getent: command not found
./netdata-uninstaller.sh: line 132: getent: command not found
./netdata-uninstaller.sh: line 141: getent: command not found
./netdata-uninstaller.sh: line 150: getent: command not found
./netdata-uninstaller.sh: line 159: getent: command not found
./netdata-uninstaller.sh: line 168: getent: command not found
```

**Comment by ktsaou** [comment]
fixed in in PR #3773
Once merged, you can re-install netdata (use netdata-updater.sh) and then the netdata-uninstaller script will be updated. Run it to uninstall netdata.

**Comment by ktsaou** [comment]
merged it.
